docs/usage/configuration-options.md

```diff
@@ -1268,10 +1268,12 @@ For example to apply a special label for Major updates:
 }
 ```
 
+Update type `patch` only matches if `separateMinorPatch` is set to true.
+
 ## patch
 
 Add to this object if you wish to define rules that apply only to patch updates.
-See also `major` and `minor` configuration options.
+Only applies if `separateMinorPatch` is set to true.
```

Comment:
I can't see any examples where we link to anchors in the documentation, but this _should_ work, and I think it makes it easier to navigate the large documenation page.

```suggestion
Only applies if [`separateMinorPatch`](#separateMinorPatch) is set to true.
```